spk/bicbucstriim/Makefile

```diff
@@ -1,18 +1,18 @@
 SPK_NAME = bicbucstriim
-SPK_VERS = 1.2.5
-SPK_REV = 4
+SPK_VERS = 1.3.6
+SPK_REV = 1
 SPK_ICON = src/bicbucstriim.png
 DSM_UI_DIR = app
 
 DEPENDS  = cross/$(SPK_NAME)
 
-MAINTAINER = Diaoul
+MAINTAINER = Alexandre Espinosa Menor
```

Comment:
MAINTAINER is the original developer of the package, or alternatively, @Diaoul and you have agreed that you'll take over the maintenance of the package in the future.